resilience4j-timelimiter/src/main/java/io/github/resilience4j/timelimiter/TimeLimiter.java

```diff
@@ -27,7 +35,18 @@ static TimeLimiter ofDefaults() {
      * @return The {@link TimeLimiter}
      */
     static TimeLimiter of(TimeLimiterConfig timeLimiterConfig) {
-        return new TimeLimiterImpl(timeLimiterConfig);
+        return of(DEFAULT_NAME, timeLimiterConfig);
+    }
+
+    /**
+     * Creates a TimeLimiter decorator with a TimeLimiterConfig configuration.
+     *
+     * @param name the String
+     * @param timeLimiterConfig the TimeLimiterConfig
+     * @return The {@link TimeLimiter}
+     */
+    static TimeLimiter of(String name, TimeLimiterConfig timeLimiterConfig) {
```

Comment:
Should I overload other constructors and factory methods (adding version with `name` parameter)?